### PR TITLE
Fixed a bug with printing of actual stats

### PR DESCRIPTION
**Pull request by pavelvelikhov**
### Changelog entry 

...

### Changelog category 

* Bugfix 

### Additional information

...

### CODE REVIEW

File: ydb/core/kqp/opt/kqp_query_plan.cpp [hunk 1]
```diff
@@ -2144,6 +2144,47 @@ double ComputeCpuTimes(NJson::TJsonValue& plan) {
...
+void ComputeTotalRows(NJson::TJsonValue& plan) {
+    
+    if (plan.GetMapSafe().contains("Plans")) {
+        for (auto& p : plan.GetMapSafe().at("Plans").GetArraySafe()) {
+            ComputeTotalRows(p);
+        }
+    }
+
+    if (plan.GetMapSafe().contains("Stats") && plan.GetMapSafe().contains("Operators")) {
+        YQL_CLOG(TRACE, CoreDq) << "Found Operators";
```
Comment (ulya-sidorina) [inline ydb/core/kqp/opt/kqp_query_plan.cpp:2156]: Let's write more understandable log message or delete it

Comment (ulya-sidorina) [inline ydb/core/kqp/opt/kqp_query_plan.cpp:2156]:
```suggestion
```

Comment (pavelvelikhov) [reply ydb/core/kqp/opt/kqp_query_plan.cpp]: deleted

Comment (pavelvelikhov) [reply ydb/core/kqp/opt/kqp_query_plan.cpp]: Deleted

**Review by ulya-sidorina — APPROVED** [review]

**Pull request merged**